src/components/Settings/Settings.scss

```
@@ -1305,6 +1305,39 @@
       scrollbar-width: thin;
     }
 
+    // Group headers ("Presets" / "My Voices" in the voice dropdown). The UA
+    // paints an <optgroup>'s `label` attribute in BLACK — 1.4:1 against the
+    // $bg-surface picker, so it reads as a missing header — and gives it no
+    // box we can align, leaving it a few px left of the option text. The
+    // customizable-select spec's stylable replacement is a <legend> child,
+    // which Chromium renders INSTEAD of the attribute; VoiceLibrarySection
+    // emits one behind supportsBaseSelect().
+    //
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)